panel/widgets/select.py

```diff
@@ -161,7 +161,7 @@ def _process_param_change(self, msg):
             if (all(isinstance(values, dict) for values in groups.values()) is False
                and  all(isinstance(values, list) for values in groups.values()) is False):
                 raise ValueError(
-                    'The values of the groups dictionnary must be all of '
+                    'The values of the groups dictionary must be all of '
                     'the dictionary or the list type.'
```

Comment:
Sorry should have actually read this. What does "all of the dictionary or list type" mean?